README.md

```diff
@@ -328,3 +328,30 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+## Generate ERD with django extensions
+```
+ pip install graphviz
+ pip install django-extensions
+ ```
+ Go to ```settings.py``` and add the following line to  ```INSTALLED_APPS```
+ ```
+ 'django_extensions',
+ ```
+
+Add the following code to ```settings.py```
+```
```

Comment:
This is already in settings.py, why are you instructing the dev to do this

Reply:
I just added it so the user knows that this specific line is essential to export db structure. But will remove it if necessary.